src/utils/buzz.ts

```diff
@@ -0,0 +1,56 @@
+export const parseBuzzTransactionDetails = (
+  details?: MixedObject | null
```

Comment:
Just for better typesafety
```suggestion
  details?: GetUserBuzzTransactionsResponse['details']
```